test/torchaudio_unittest/datasets/utils_test.py

```diff
@@ -1,14 +1,23 @@
-from torchaudio.datasets import utils as dataset_utils
-from torchaudio.datasets.commonvoice import COMMONVOICE
-
 from torchaudio_unittest.common_utils import (
-    TempDirMixin,
     TorchaudioTestCase,
     get_asset_path,
 )
 
+from torchaudio.datasets import utils as dataset_utils
+from torchaudio.datasets.commonvoice import COMMONVOICE
+
+original_ext_audio = COMMONVOICE._ext_audio
+
 
 class TestIterator(TorchaudioTestCase):
+    @classmethod
+    def setUpClass(cls):
+        COMMONVOICE._ext_audio = ".wav"
+
+    @classmethod
+    def tearDownClass(cls):
+        COMMONVOICE._ext_audio = original_ext_audio
```

Comment:
Looks good, but this makes me think that we should not be using CommonVoice dataset implementation in this test.

This is not the scope of this PR but we should define an trivial, dedicated Dataset for this utility test and stop using CommonVoice here. Then we can finally remove the `mp3` asset `CommonVoice/cv-corpus-4-2019-12-10/tt/clips/common_voice_tt_00000000.mp3`.